### ISSUE TITLE
record.ttl() invalid value

### ISSUE THREAD

**Issue by erikdubbelboer** [aerospike/aerospike-lua-core#9]
The `record.ttl()` function has an integer overflow bug for expired keys. When a key has expired but is still present the ttl is a negative value but this somehow overflows and results in super hight ttl numbers.

Below I included a Go program to replicate this issue. The output will look as follows:
```
$ go run bug.go
5
4294967291
```

In the aerospike.log it will print:
```
Nov 30 2015 07:56:39 GMT: DEBUG (udf): ([C]::-1) ttl: 5
Nov 30 2015 07:56:49 GMT: DEBUG (udf): ([C]::-1) ttl: 4294967291
```

```go
package main

import (
  "fmt"
  "time"

  "github.com/aerospike/aerospike-client-go"
)

func main() {
  client, err := aerospike.NewClient("127.0.0.1", 3000)
  if err != nil {
    panic(err)
  }

  key, err := aerospike.NewKey("test", "test", "test")
  if err != nil {
    panic(err)
  }

  script := `
function foo(r)
  local ttl = record.ttl(r)

  debug("ttl: %s", tostring(ttl))

  return ttl
end
`

  if task, err := client.RegisterUDF(nil, []byte(script), "test.lua", aerospike.LUA); err != nil {
    panic(err)
  } else {
    for err := range task.OnComplete() {
      if err != nil {
        panic(err)
      }
    }
  }

  wp := aerospike.NewWritePolicy(0, 0)
  wp.Expiration = 5 // Expire in 5 seconds.

  bin := aerospike.NewBin("test", 1)

  if err := client.PutBins(wp, key, bin); err != nil {
    panic(err)
  }

  if r, err := client.Execute(wp, key, "test", "foo"); err != nil {
    panic(err)
  } else {
    fmt.Printf("%#v\n", r)
  }

  time.Sleep(time.Second * 10)

  if r, err := client.Execute(wp, key, "test", "foo"); err != nil {
    panic(err)
  } else {
    fmt.Printf("%#v\n", r)
  }
}
```

**Comment by bwhisler** [comment]
Hi Erik,

This issue was fixed in server version 3.6.0.  Are you using an older version?  As of 3.6.0, record.ttl() should return zero for expired or missing records.

Thanks,

Brad

**Comment by erikdubbelboer** [comment]
I am indeed still using 3.5.15, sorry I didn't think of testing on the latest version first.
